attributes/default.rb

```diff
@@ -19,3 +19,12 @@
 #
 
 default['ms_dotnet']['timeout'] = 600
+
+if platform? 'windows'
+  nt_version = node['platform_version'].to_f
+  if nt_version >= 6.2
+    default['ms_dotnet']['feature']['enable_all']            = true
```

Comment:
Thanks I appreciate; I'll test your MR tomorrow on a 2012R2 box.
Do you have a core box to check if it's compatible? I think the core version will be a bit more trickier, but we could skip it currently.